### PR TITLE
Merge to "master"

### PR DESCRIPTION
**Pull request by rajeevbbqq**

### CODE REVIEW

File: cdk/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "typescript": "~3.9.7"
   },
   "dependencies": {
-    "@aws-cdk/aws-cloudfront-origins": "^1.152.0",
+    "@aws-cdk/aws-cloudfront-origins": "^1.138.0",
```
Comment (anandrgitnirman) [inline cdk/package.json:25]: we are we downgrading this ?

File: src/components/snet-wallet-connector/walletHook.js [hunk 2]
```diff
@@ -224,7 +224,7 @@ export const useWalletHook = () => {
       return response;
     } catch (error) {
       console.log('Approve spender error: ', error);
-      throw formatContractExceptionMessage(error);
+      throw error?.message ?? JSON.stringify(error);
```
Comment (anandrgitnirman) [inline src/components/snet-wallet-connector/walletHook.js:227]: seemed like we were formatting this before but not now?

**Review by anandrgitnirman — APPROVED** [review]
Approving based on testinf done in Ropsten 
@rajeevbbqq , can you please respond to the comments + also get a review from @ksridharbabuus

File: cdk/package.json (hunk 1 above)
Comment (Karthi96) [reply cdk/package.json:25]: yeah, bro, I made this change. you can ignore we are not using this package as i have reverted my changes

File: src/components/snet-wallet-connector/walletHook.js (hunk 2 above)
Comment (rajeevbbqq) [reply src/components/snet-wallet-connector/walletHook.js:227]: For Authorize button we only display error context

**Pull request merged**